### DISCUSSION TITLE
The Countdown Clock

### DISCUSSION THREAD

**Discussion by meviru** [meviru/badge-hunting#18] [Galaxy Trivia] [answered]
Your code wants to count down from 10 to 1 with a dramatic one-second pause between each number. How can it put on this suspenseful show using JavaScript?

**Comment by bicced** [comment]
use timeout

**Reply by meviru** [reply]
@bicced Please elaborate on how we can achieve it with the code.

**Comment by mitchiemt11** [comment, marked as answer]
I'd use `setInterval` to decrement and log the countdown value every 1000 milliseconds, clearing the interval when it reaches 0.

**Reply by meviru** [reply]
That is correct!! 👏

**Comment by PranjalTrivedi** [comment]
Mark Answer

**Comment by marstrx** [comment]
nice one
